debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -591,13 +591,13 @@ alterView
 
 alterSpecification
     : tableOption (','? tableOption)*                               #alterByTableOption
-    | ADD COLUMN? uid columnDefinition (FIRST | AFTER uid)?         #alterByAddColumn
+    | ADD COLUMN? ifNotExists? uid columnDefinition (FIRST | AFTER uid)?         #alterByAddColumn // ifNotExists is MariaDB-specific
     | ADD COLUMN?
         '('
           uid columnDefinition ( ',' uid columnDefinition)*
         ')'                                                         #alterByAddColumns
-    | ADD indexFormat=(INDEX | KEY) uid? indexType?
-      indexColumnNames indexOption*                                 #alterByAddIndex
+    | ADD indexFormat=(INDEX | KEY) ifNotExists? uid? indexType?
+      indexColumnNames indexOption*                                 #alterByAddIndex // ifExists is MariaDB-specific
```

Comment:
The comment here and below is not technically correct as `ifNotExists` is added but caomment says `ifExists`.